firedrake/mesh.py

```diff
@@ -448,6 +448,18 @@ def init(self):
         if hasattr(self, '_callback'):
             self._callback(self)
 
+    def create_cell_node_list(self, global_numbering, fiat_element, dofs_per_cell):
```

Comment:
This method is moved up in the class hierarchy, so that the sibling method can be (and is) removed.